Discard my Transition Paths

noracato opened the pull request:
The same discard functionality we have for saved scenarios is now also available for multi year charts.

Two questions remain open:
- Do we want to add a discard option to the API? Or keep just keep the hard delete like we have now?
- Do we want to show in the MYC app that this scenario is in the trash? This could be quite some work.

An impression of the trash can, which is now shared between scenarios and TP's:

![Screenshot 2023-03-24 at 14 22 48](https://user-images.githubusercontent.com/14875123/227544017-594fd4d0-32d3-4948-8504-3fb2a65e95dd.png)

---

Closes #4084

mabijkerk commented:
Useful new feature @noracato! Below my feedback (somewhat broader scope than just the delete MYC feature, hopefully that's okay).

**General feedback**
<img width="1015" alt="Screenshot 2023-03-24 at 17 08 22" src="https://user-images.githubusercontent.com/67338510/227580293-5ccc590f-c5ab-4548-8a7d-b88e0aeda295.png">

- The save icon of the "Save current scenario" could be clearer and align more with the other icons: the other icons are line-icons, while this one is a filler-icon
- I might add a "Save scenario" text to the save icon: the other tabs in "My Scenarios" all have an icon and with texts, so it seems counterintuitive that save scenario does not have text
- I'm not sure which scenario I'm saving here: I'm accessing "My scenarios" from the homepage, so I don't have any scenario open yet. Why do I get the save scenario option?
- I think I would prefer it if each box would have the same size, currently "My Transition Paths" is broader than "Trash"

**Delete MYC feedback**

- I don't see an icon appearing over an MYC scenario when I hover above it with my mouse, like it does for the regular scenarios. I'm therefore not sure how to actually delete an MYC scenario
- For the trash can, it might be useful if the scenario information has a type that says "my scenarios" or "my transition paths". I find it difficult to tell the difference only by the two different icons

noracato commented:
Thanks for your feedback @mabijkerk!

> For the trash can, it might be useful if the scenario information has a type that says "my scenarios" or "my transition paths". I find it difficult to tell the difference only by the two different icons

I changed the label styles in the trash to be more clear:

![Screenshot 2023-03-27 at 12 05 41](https://user-images.githubusercontent.com/14875123/227911143-8230b384-759f-4e62-955c-b33f7c77f975.png)

> I don't see an icon appearing over an MYC scenario when I hover above it with my mouse, like it does for the regular scenarios. I'm therefore not sure how to actually delete an MYC scenario

I can't reproduce this - with you it also works now, right?

> The save icon of the "Save current scenario" could be clearer and align more with the other icons: the other icons are line-icons, while this one is a filler-icon
I might add a "Save scenario" text to the save icon: the other tabs in "My Scenarios" all have an icon and with texts, so it seems counterintuitive that save scenario does not have text

This is only for admins. Because we have an extra tab, the original button that says 'save current scenario' does not fit. Hence I replaced it with an icon with a tooltip with the original text last week. Normal users are not affected.

> I'm not sure which scenario I'm saving here: I'm accessing "My scenarios" from the homepage, so I don't have any scenario open yet. Why do I get the save scenario option?

Could be a scenario you accessed hours before. I think we keep the session alive for quite a while.

> I think I would prefer it if each box would have the same size, currently "My Transition Paths" is broader than "Trash"

Personally I don't mind it. But this would require a small redesign of the whole page because it simply does not fit 😅
